mockk/jvm/src/test/kotlin/io/mockk/it/MockStaticMethodTest.kt

```diff
@@ -2,13 +2,43 @@ package io.mockk.it
 
 import io.mockk.every
 import io.mockk.mockkStatic
+import io.mockk.unmockkStatic
 import io.mockk.verify
+import org.junit.Test
+import java.time.Instant
 import kotlin.test.Test
 import kotlin.test.assertEquals
+import kotlin.test.assertNotEquals
 
 val Int.selfOp get() = this * this
 
-class StaticMockJVMTest {
+class MockStaticMethodTest {
+    /**
+     * github issue #92
+     */
+    @Test
+    fun test() {
```

Comment:
I'd suggest using a better name on tests, describing what the test is actually doing.

Reply:
Thank you for suggest, I didn't change method name because it was not required for the issue, but i will.